<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackendGuIClass</class>
 <widget class="QMainWindow" name="BackendGuIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BackendGuI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>35</y>
      <width>133</width>
      <height>167</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="yearPlanBtn">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>年度计划</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="seasonPlanBtn">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>季节计划</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="monthPlanBtn">
       <property name="minimumSize">
        <size>
         <width>131</width>
         <height>51</height>
        </size>
       </property>
       <property name="text">
        <string>月计划</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>292</x>
      <y>241</y>
      <width>182</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>216</y>
      <width>184</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="accountEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>41</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>41</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="pwdEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>158</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="confirmBtn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>计划管理器 HL1.0</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHL1_0">
    <property name="title">
     <string>     </string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>   </string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuHL1_0"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>f</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BackendGuI.qrc"/>
 </resources>
 <connections/>
</ui>
